### DISCUSSION TITLE
Multi-language support for messages

### DISCUSSION THREAD

**Discussion by cristianoag** [cristianoag/microsoft-teams-apps-company-communicator#16] [Ideas]
Ability to create adaptive cards (notifications) in multiple languages and have those delivered according to the language defined for the Teams client.

**Comment by SammyRasha** [comment]
https://adaptivecards.io/blog/2021/Announcing-1.5/

Right to Left text support was recently added to Adaptive Cards -- should make implementation somewhat easier
